bi_sql_editor/views/view_bi_sql_view.xml

```diff
@@ -123,50 +120,42 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                             <field name="ttype" />
                             <field
                                 name="many2one_model_id"
-                                attrs="{
-                                'invisible': [('ttype', '!=', 'many2one')],
-                                'required': [('ttype', '=', 'many2one')],
-                            }"
+                                invisible="ttype != 'many2one'"
```

Comment:
@nguyenminhchien in this context of the tree view, I think we would use `column_invisible`

Reply:
@dzungtran89 `column_invisible` is useful when its value is "1" or base on fields from parent, such as                                 `column_invisible="not parent.option_context_field"`

Reply:
indeed, just the matter of `invisibile` meaning, the column is still visible, but the data is hidden; while the `column_*` hides the column

Reply:
yes, so, we should keep "invisible" in this case. for example, our tree has 2 rows:
- Row 1: ttype=many2one  ==> column_invisible=True
- Row 2: ttype=float  ==> column_invisible=False
Attribute "column_invisible" will be confused and useless.